demo/build.gradle

```diff
@@ -88,18 +92,24 @@ def createTask(node) {
         dependsOn { project(":${node.adapter}").shadowJar }
         dependsOn { project(":node").bootJar }
 
-        from(project(":${node.adapter}").shadowJar)
-        from("${project(':node').buildDir}/libs")
+        into "${buildDir}/demo/${node.name}"
```

Comment:
Could we start splitting these copy tasks into separate tasks for each subfolder or something similar - since this task is getting bigger and bigger.